README.md

```diff
@@ -16,11 +16,10 @@
 
 # 📣 What does this dbt package do?
 
-This package models AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/applications/aws_cloud_cost). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/aws_cloud_cost#schemainformation).
+This package models AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/aws-cost-report). 
```

Comment:
The connector link is broken and needs to be updated. Also, there is no ERD for this connector so we need to update this part of the README to properly inform customers.

Reply:
adjusted this somewhat but will circle back when polishing docs and such